examples/pxScene2d/src/pxContextGL.cpp

```diff
@@ -1132,11 +1141,20 @@ void linkShaderProgram(GLuint program)
 class shaderProgram
 {
 public:
-  virtual ~shaderProgram() {}
+  virtual ~shaderProgram() {
+   glDetachShader(mProgram, mFragShader);
+   glDetachShader(mProgram, mVertShader);
+   glDeleteShader(mFragShader);
+   glDeleteShader(mVertShader);
+   glDeleteProgram(mProgram);
+  }
   virtual void init(const char* v, const char* f)
   {
-    mProgram = createShaderProgram(v, f);
-
+    glShaderProgDetails details = {};
```

Comment:
You could skip this empty declaration / assignment by...

`glShaderProgDetails  details = createShaderProgram(v, f);`